.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: scala
 scala:
-  - 2.12.9
+  - 2.12.10
+  - 2.13.1
 jdk:
   - openjdk8
 script:
-  - sbt -S-Xfatal-warnings ++$TRAVIS_SCALA_VERSION clean coverage test
```

Comment:
Why aren't we running fatal-warnings in the CI anymore?

Reply:
Even without the `-deprecation` flag, we still get a warning that there are deprecations, so we can't turn all warnings into errors. We can, however, enable fatal warnings for Scala 2.12. I did that in adfa9a2.